closure/goog/events/inputhandler.js

```diff
@@ -81,7 +81,7 @@ goog.events.InputHandler = function(element) {
   //   event.
   // IE9 supports input events when characters are inserted, but not deleted.
   // WebKit before version 531 did not support input events for textareas.
-  var emulateInputEvents = goog.userAgent.IE ||
+  var emulateInputEvents = goog.userAgent.IE || goog.userAgent.EDGE ||
```

Comment:
I'm not sure that's right given the comment in the header:

```
 *     IE doesn't have native support for input event. WebKit before version 531
 *     doesn't have support for textareas. For those browsers an emulation mode
 *     based on key, clipboard and drop events is used. Thus this event won't
 *     trigger in emulation mode if text was modified by context menu commands
 *     such as 'Undo' and 'Delete'.
```

Seems to say that emulation is only on for those specific browsers (IE [and Edge], WebKit before 531)... so the code seems clear in that case?

I guess that can be EDGE_OR_IE to save some characters?